LIMS2DB/classes.py

```diff
@@ -52,6 +52,10 @@ def __init__(self, lims, crawler, log):
                 pjs[p.id]['library'] = p.udf['Library construction method']
             except KeyError:
                 pjs[p.id]['library'] = None
+            try:
+                pjs[p.id]['library_option'] = p.udf['Library prep option']
+            except KeyError:
+                pjs[p.id]['library_option'] = None
```

Comment:
Do we need to have a try except here? Can't it just be
```suggestion
            pjs[p.id]['library_option'] = p.udf.get('Library prep option', None)
```
I haven't tested it though